integration/oasis/stationgraph/node.go

```diff
@@ -51,8 +51,7 @@ func (n *node) calcChargeTime(prev *node, distance float64, strategy chargingstr
 	if arrivalEnergy < 0 {
 		glog.Fatalf("Before updateNode should check isLocationReachable() prev.arrivalEnergy=%#v distance=%#v", prev.arrivalEnergy, distance)
 	}
-	//return strategy.EvaluateCost(arrivalEnergy, chargingstrategy.ChargingStatus{ChargingEnergy: n.chargeEnergy}).Duration
-	return 0.0
+	return strategy.EvaluateCost(arrivalEnergy, chargingstrategy.State{ChargingEnergy: n.chargeEnergy}).Duration
```

Comment:
There is an update suggestion here:
https://github.com/Telenav/osrm-backend/issues/211#issuecomment-597239787